### PR TITLE
fix(pfp): feature flag for "step two" of PFP

### LINKED ISSUE #1884: [M1] Step Two in "Link Bank Account" should be disabled / sealed off
Currently in the 1.25 release, the step 2 button in the "link bank account" flow is accessible when the user's KYC status is "completed". Really this should be "approved", which Joe recently fixed, but also it should not be accessible at all while we are still developing the Plaid flow. Our contract with them hasn't started yet, so it would be especially bad to start hitting the plaid endpoints.

AC:
- feature flag on Alfajores and Mainnet to seal off M2 for all users

### PR DESCRIPTION
**Pull request by cajubelt**
### Description

Currently "step 2" of the link bank account flow is not sealed off by default for users who have completed KYC with Persona. This is a problem because the Plaid flow is still under development.

This adds a feature flag to make sure the button is disabled until we are done with step 2 of the PFP (Persona, Finclusive, Plaid) integration.

### Other changes

na

### Tested

added automated test coverage

manual test looks good. With firebase remote config `linkBankAccountStepTwoEnabled` turned off:
simulator_screenshot_DABFECDB-42F6-47CC-9CC7-5B4DDDD63165

and with `linkBankAccountStepTwoEnabled` turned on:
simulator_screenshot_EA70F8F9-16E4-4109-BD1D-1883ADC93177

### How others should test

Since connecting your bank account is not complete, you should expect the "step two" section of the "link your bank account" settings page to have a button that is disabled, even after you verify your identity with Persona.

To test:
- with an account on alfajores with a verified phone number:
- go to settings
- tap "Link Bank Accounts"
- complete Persona flow ("step 1")
- verify that "coming soon!" button is disabled (greyed out and tapping it does nothing)

### Related issues

- Fixes #1884 

### Backwards compatibility

yes

### CODE REVIEW

**Comment by cajubelt** [comment]
branched off https://github.com/valora-inc/wallet/pull/1877, which can be merged first

File: packages/mobile/test/schemas.ts [hunk 1]
```diff
@@ -930,6 +930,7 @@ export const v31Schema = {
     biometryEnabled: false,
     supportedBiometryType: null,
     useBiometry: false,
+    linkBankAccountStepTwoEnabled: false,
```
Comment (cajubelt) [inline packages/mobile/test/schemas.ts:933]: can someone weigh in on whether this is in the right place or needs a v32Schema? My assumption is that we only need 1 new schema per app release, but I'm not actually sure about that

Comment (jh2oman) [reply packages/mobile/test/schemas.ts:933]: I am also not sure about when upgrades are needed. May need to ask the retention squad

**Review by lisaqyli — APPROVED** [review]

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified the above task on Latest Android Alfajores Build V:1.26.0 & updated the status in test rails.

**Test rail link:**
**Android 8:** https://valoraapp.testrail.io/index.php?/tests/view/9818&group_by=cases:section_id&group_order=asc&group_id=35

**Android 10:** https://valoraapp.testrail.io/index.php?/tests/view/9944&group_by=cases:section_id&group_order=asc&group_id=35

**Observation :**
1. Few components are not seen on Android application as compared with iOS and we have raised a [bug](https://github.com/valora-inc/wallet/issues/1915) for it.

2. "Get Access" is seen for link bank account section even after the Phone number is already verified after user restores the account and we have raised a [bug](https://github.com/valora-inc/wallet/issues/1886) for it.

**Devices** : Samsung Galaxy A8+(8.0), Samsung Galaxy M40(10.0)

**Cc:** @cajubelt 
Thanks..!!

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified the above task on Latest Android Alfajores Build and Latest iOS Alfajores build V:1.26.0 & updated the status in test rails. 

**Observation :** 
- Few components are not seen on Android application as compared with iOS
-"Get Access" is seen for link bank account section even after the Phone number is already verified after user restores the account
- Link bank account is getting enabled after completing the first step of verification for both Android and iOS app
 
**Attachment :** [enable](https://drive.google.com/file/d/1Sn650Hm5bKvBhx_Vw2oAim5TDBpm35MZ/view?usp=sharing)


**Test rail link:**
**Android 11**: https://valoraapp.testrail.io/index.php?/tests/view/9818&group_by=cases:section_id&group_order=asc&group_id=35

**iOS 14:** https://valoraapp.testrail.io/index.php?/tests/view/9944&group_by=cases:section_id&group_order=asc&group_id=35

Bugs :
https://github.com/valora-inc/wallet/issues/1915
https://github.com/valora-inc/wallet/issues/1886

Devices : Google Pixel 2XL(11.0) , iPhone 12(14.7.1)

Cc: @cajubelt
Thanks..!!

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified the above task on Latest iOS Alfajores build V:1.26.0 & updated the status in test rails.

**Observation :**

- "Get Access" is seen for link bank account section even after the Phone number is already verified after user restores the account
- Link bank account is getting enabled after completing the first step of verification for both Android and iOS app

**Attachment:** [enable.mp4](https://drive.google.com/file/d/1Sn650Hm5bKvBhx_Vw2oAim5TDBpm35MZ/view?usp=sharing)

**Test rail link:**
**iOS 13:** https://valoraapp.testrail.io/index.php?/tests/view/10010&group_by=cases:section_id&group_order=asc&group_id=35

**Bug:** [#1886](https://github.com/valora-inc/wallet/issues/1886)

**Devices:** iPhone 7(13.3)

**CC:** @cajubelt 

Thanks..!!

**Pull request merged**